### PR TITLE
Documentation edits

### PR DESCRIPTION
**Pull request by kontsevoy**
Some users noticed that 'display' field is not well-documented for the
connectors.

I also noticed that some defaults are not sensible (like "google" as the
provider)

### CODE REVIEW

File: examples/resources/adfs-connector.yaml [hunk 1]
```diff
@@ -5,6 +5,14 @@ version: v2
 metadata:
   name: "adfs_connector"
 spec:
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "login with SSO"
```
Comment (russjones) [inline examples/resources/adfs-connector.yaml]: Capital L, `Login with SSO`? Or maybe `Login with ADFS`, but I don't know Windows applications refer to it as SSO or ADFS?

File: examples/resources/saml-connector.yaml [hunk 2]
```diff
@@ -8,6 +8,9 @@ metadata:
   # the name of the connector
   name: new_saml_connector
 spec:
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "login with SSO"
```
Comment (russjones) [inline examples/resources/saml-connector.yaml]: Capital L: `Login with SSO`?

File: docs/2.3/saml.md [hunk 3]
```diff
@@ -81,7 +81,11 @@ version: v2
 metadata:
   name: OktaSAML
 spec:
-  acs: https://localhost:3080/v1/webapi/saml/acs
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "Login with Okta SSO"
+
```
Comment (russjones) [inline docs/2.3/saml.md]: `acs is the Assertion Consumer Service URL. This should be the address of the Teleport proxy that your identity provider will communicate with.`

File: examples/resources/saml-connector.yaml (hunk 2 above)
Comment (russjones) [inline examples/resources/saml-connector.yaml]: Capital L, `Login with SSO`?

**Review by russjones — APPROVED** [review]

File: docs/2.3/saml.md (hunk 3 above)
Comment (kontsevoy) [reply docs/2.3/saml.md]: oh, nice. I'll add that, thanks.

File: examples/resources/adfs-connector.yaml [hunk 4]
```diff
@@ -5,6 +5,14 @@ version: v2
 metadata:
   name: "adfs_connector"
 spec:
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "Login with SSO"
+
+  # "adfs" provider setting tells Teleport that this SAML connector uses ADFS
+  # as a provider
```
Comment (russjones) [inline examples/resources/adfs-connector.yaml]: This field is required for ADFS to override the default C14N11 canonicalizer and use the C14N10 canonicalizer.

**Pull request merged**